src/main/java/org/zone/commands/structure/zone/flags/greetings/SetGreetingsMessageCommand.java

```diff
@@ -0,0 +1,70 @@
+package org.zone.commands.structure.zone.flags.greetings;
+
+import net.kyori.adventure.text.Component;
+import org.spongepowered.api.command.CommandCause;
+import org.spongepowered.api.command.CommandResult;
+import org.spongepowered.configurate.ConfigurateException;
+import org.zone.commands.system.ArgumentCommand;
+import org.zone.commands.system.CommandArgument;
+import org.zone.commands.system.NotEnoughArgumentsException;
+import org.zone.commands.system.arguments.operation.ExactArgument;
+import org.zone.commands.system.arguments.operation.RemainingArgument;
+import org.zone.commands.system.arguments.simple.StringArgument;
+import org.zone.commands.system.arguments.zone.ZoneArgument;
+import org.zone.commands.system.context.CommandContext;
+import org.zone.region.Zone;
+import org.zone.region.flag.FlagTypes;
+import org.zone.region.flag.greetings.GreetingsFlag;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+public class SetGreetingsMessageCommand implements ArgumentCommand {
+    public static final ExactArgument ZONE = new ExactArgument("zone");
+    public static final ExactArgument FLAGS = new ExactArgument("flags");
+    ZoneArgument ZONE_VALUE = new ZoneArgument("zone_value", new ZoneArgument.ZoneArgumentPropertiesBuilder());
```

Comment:
Why is this argument not like the rest?

Reply:
oh cuz u told?

Reply:
that's a zone argument

Reply:
Im aware its a zone argument. I mean why does it not have ``public static final`` like the rest?